gpMgmt/sbin/packcore

```
@@ -27,13 +27,17 @@ def _getPlatformInfo():
 
 
 def _getFileInfo(coreFile):
-    cmd = Popen(['/usr/bin/file', '--version'], stdout=PIPE, stderr=STDOUT)
+    file_cmd = which('file')
+    if file_cmd is None:
+        return False
```

If I am not wrong this function should return a string instead of bool. I think we should raise an exception if which('file') is None. thoughts?

---

i just imitate the ` _getLibraryListWithGDB` in this file, but i also think raising an error is better.